src/InputBox.tsx

```diff
@@ -244,7 +244,7 @@ const flagMap = new Map([
   ["🇹🇿", "Tanzania"],
   ["🇺🇦", "Ukraine"],
   ["🇺🇬", "Uganda"],
-  ["🇺🇸", "United States"],
+  ["🇺🇸", "USA"],
```

Comment:
this standardizes this somehow. i think for country codes which was an old direction. i dont care enough to remove this change! let it be!